src/main/java/frc/team166/robot/subsystems/Lift.java

```diff
@@ -324,6 +360,46 @@ protected boolean isFinished() {
         };
     }
 
+    public Command LowerLiftToLimitSwitch() {
+        return new SubsystemCommand(this) {
+            @Override
+            protected void initialize() {
+                disengageBrake();
+            }
+
+            @Override
+            protected void execute() {
+                lowerLift();
+            }
+
+            @Override
+            protected boolean isFinished() {
+                return !bottomLimitSwitch.get();
+            }
+        };
+    }
+
+    public Command LowerLiftForTime() {
```

Comment:
This should probably accept a double timeout as an argument and use that for the setTimeout() call.

Reply:
done